controllers/tc000230_drift_detection_only_mode_test.go

```diff
@@ -0,0 +1,242 @@
+package controllers
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	. "github.com/onsi/gomega"
+
+	infrav1 "github.com/chanwit/tf-controller/api/v1alpha1"
+	sourcev1 "github.com/fluxcd/source-controller/api/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// +kubebuilder:docs-gen:collapse=Imports
+
+func Test_000230_drift_detection_only_mode(t *testing.T) {
+	Spec("This spec describes setting drift detection only mode which will skip the plan and apply stages")
+
+	const (
+		sourceName    = "test-tf-drift-detection-only-mode"
+		terraformName = "helloworld-no-outputs"
```

Comment:
Please change these 2 lines to
```
               sourceName    = "gr-drift-detection-only-mode"
               terraformName = "tf-drift-detection-only-mode"
```
as one of them conflicted with another test.

Reply:
Thanks for spotting.